### PR TITLE
Feature/fallback image

### PR DESCRIPTION
**Pull request by tuj**
Adds fallback image to activities

### CODE REVIEW

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 1]
```diff
@@ -63,8 +63,9 @@ function genlyd_aarhus_preprocess_node__activity(&$variables) {
   $node = $variables['node'];
 
   // Get categories.
+  $referencedCategories = $node->field_categories->referencedEntities();
   $categories = [];
-  foreach ($node->field_categories->referencedEntities() as $category) {
+  foreach ($referencedCategories as $category) {
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Variable  $referencedCategories only used once?

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 2]
```diff
@@ -338,3 +327,42 @@ function _genlyd_aarhus_get_price_text($price) {
...
+        $image = $category->field_default_image->entity->getFileUri();
+        $imageUrl = ImageStyle::load($imageStyle)->buildUrl($image);
+        break;
+      }
+    }
+
+    // If no image has been set, select placeholder image.
+    if (!isset($imageUrl)) {
+      // Load category default image.
+      $imageUrl = "/themes/custom/genlyd_aarhus/source/assets/images/styleguide/placeholder-hero.png";
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Use drupal_get_path('theme', 'genlyd_aarhus').

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme [hunk 3]
```diff
@@ -338,3 +327,42 @@ function _genlyd_aarhus_get_price_text($price) {
     return \Drupal::translation()->translate(':price kr.', [':price' => $price]);
   }
 }
+
+/**
+ * Get the image url for the activity.
+ *
```
Comment (cableman) [inline web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme:332]: Little more description:

Something like:
>If activtive don't have image, trying to get it from it's selected categories and finally fallback to theme provide default image.

**Review by cableman — CHANGES_REQUESTED** [review]

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme (hunk 1 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Done.

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme (hunk 3 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme:332]: Done.

File: web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme (hunk 2 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme]: Done.

**Review by cableman — APPROVED** [review]

**Pull request merged**